html-web-page/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock jelly!</h2>
+  <p>
+    Here at JOTMC Inc., we are constantly striving to supply our customers with
+    the highest-quality and most unique jellies on a monthly basis. For the jelly
+    lover in your life (or yourself!), this is the gift that keeps on giving all
+    year round. We offer three tiers of monthly jelly joy, and each one offers their
+    own perks. Here is a breakdown of each, as well as your monthly price:
+  </p>
+
+  <!--table code-->
+  <table border="2">
+    <thead>
+      <th></th>
+      <th>Basic Plan</th>
+      <th>Deluxe Plan</th>
+      <th>Jelly Giant Plan</th>
```

Comment:
These should be in a `tr`

Reply:
But aren't they table heading elements, if I change them to tr elements they would be just like the rest of the data entries?